blob/fileblob/fileblob_test.go

```diff
@@ -36,14 +36,23 @@ import (
 )
 
 type harness struct {
-	dir       string
-	prefix    string
-	server    *httptest.Server
-	urlSigner URLSigner
-	closer    func()
+	dir          string
+	prefix       string
+	skipMetadata bool
+	server       *httptest.Server
+	urlSigner    URLSigner
+	closer       func()
 }
 
-func newHarness(ctx context.Context, t *testing.T, prefix string) (drivertest.Harness, error) {
+func newHarness(ctx context.Context, t *testing.T, prefix string, skipMetadata bool) (drivertest.Harness, error) {
+	switch name := t.Name(); {
+	case strings.HasSuffix(name, "TestAttributes"), strings.Contains(name, "TestMetadata/"):
```

Comment:
Nice, but can you add a comment here explaining that we're skipping testcases that verify metadata? I'm a bit surprised it's only these two :-|.